### PR TITLE
media-video/gaupol: bump to 0.91, update LICENSE, HOMEPAGE

### PR DESCRIPTION
**Pull request by sbraz**
I took the liberty of replacing `fdo-utils` calls with `xdg`, update the LICENSE to reflect upstream's and only display postinst messages on the first merge.
https://github.com/otsaloma/gaupol/releases/tag/0.91
```diff
--- gaupol-0.28.2-r1.ebuild	2016-08-05 14:38:07.290679755 +0200
+++ gaupol-0.91.ebuild	2016-08-05 14:38:05.820725807 +0200
@@ -2,19 +2,17 @@
 # Distributed under the terms of the GNU General Public License v2
 # $Id$
 
-EAPI=5
+EAPI=6
 
 PYTHON_COMPAT=( python{3_3,3_4,3_5} )
 
-inherit distutils-r1 fdo-mime gnome2-utils versionator virtualx
-
-MAJOR_MINOR_VERSION="$(get_version_component_range 1-2)"
+inherit distutils-r1 gnome2-utils virtualx xdg
 
 DESCRIPTION="A subtitle editor for text-based subtitles"
 HOMEPAGE="http://otsaloma.io/gaupol/"
-SRC_URI="http://download.gna.org/${PN}/${MAJOR_MINOR_VERSION}/${P}.tar.xz"
+SRC_URI="https://github.com/otsaloma/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
 
-LICENSE="GPL-2"
+LICENSE="GPL-2+"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
 IUSE="spell test"
@@ -38,19 +36,26 @@
 
 DOCS=( AUTHORS.md NEWS.md TODO.md README.md README.aeidon.md )
 
+src_prepare() {
+	xdg_src_prepare
+}
+
 pkg_preinst() {
+	xdg_pkg_preinst
 	gnome2_icon_savelist
 }
 
 pkg_postinst() {
-	fdo-mime_desktop_database_update
-	fdo-mime_mime_database_update
+	xdg_pkg_postinst
 	gnome2_icon_cache_update
-	elog "Previewing support needs MPlayer or VLC."
-
-	if use spell; then
-		elog "Additionally, spell-checking requires a dictionary, any of"
-		elog "Aspell/Pspell, Ispell, MySpell, Uspell, Hspell or AppleSpell."
+	
+	if [[ -z "${REPLACING_VERSIONS}" ]]; then
+		elog "Previewing support needs MPV, MPlayer or VLC."
+
+		if use spell; then
+			elog "Additionally, spell-checking requires a dictionary, any of"
+			elog "Aspell/Pspell, Ispell, MySpell, Uspell, Hspell or AppleSpell."
+		fi
 	fi
 }
 
@@ -59,7 +64,6 @@
 }
 
 pkg_postrm() {
-	fdo-mime_desktop_database_update
-	fdo-mime_mime_database_update
+	xdg_pkg_postrm
 	gnome2_icon_cache_update
 }


```

### CODE REVIEW

**Comment by EvaSDK** [comment]
@gentoo/python @gentoo/video

File: media-video/gaupol/gaupol-0.91.ebuild [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+	test? (
+		dev-python/pytest[${PYTHON_USEDEP}]
+		dev-python/pytest-runner[${PYTHON_USEDEP}]
+	)
+"
+
+DOCS=( AUTHORS.md NEWS.md TODO.md README.md README.aeidon.md )
+
+src_prepare() {
+	xdg_src_prepare
```
Comment (mgorny) [inline media-video/gaupol/gaupol-0.91.ebuild]: It's generally a bad idea to override distutils-r1 phases.

File: media-video/gaupol/gaupol-0.91.ebuild [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+}
+
+pkg_preinst() {
+	xdg_pkg_preinst
+	gnome2_icon_savelist
+}
+
+pkg_postinst() {
+	xdg_pkg_postinst
```
Comment (mgorny) [inline media-video/gaupol/gaupol-0.91.ebuild]: You could just call `xdg_desktop_database_update` and `xdg_mimeinfo_database_update` from xdg-utils. That would save you from having `pkg_preinst()` whose only purpose is guessing waht to call in postinst.

Comment (mgorny) [inline media-video/gaupol/gaupol-0.91.ebuild]: (and also get rid fo the crappy xdg.eclass)

File: media-video/gaupol/gaupol-0.91.ebuild (hunk 1 above)
Comment (floppym) [inline media-video/gaupol/gaupol-0.91.ebuild]: To expand on that: `distutiils-r1_src_configure` already calls `xdg_environment_reset`, making xdg_src_prepare entirely redundant.

**Comment by floppym** [comment]
lgtm

**Pull request closed without merging**